packages/cli/src/commands/join.ts

```diff
@@ -383,24 +451,36 @@ function iteratePotentialConflicts(potentialConflicts: any) {
             }
           }
         } else {
-          showConflicts(green(group) +' => '+ key, conflicts);
-          potentialConflictsTotal += conflicts.length;
+          showConflicts(green(group) + ' => ' + key, conflicts);
+          if (!skipTagsCheck) {
+            potentialConflictsTotal += conflicts.length;
+          }
+        }
+        if (group === 'tags') {
+          prefixTagSuggestion(conflicts.length, skipTagsCheck);
         }
-        prefixTagSuggestion(group, conflicts.length);
       }
     }
   }
 }
 
-function prefixTagSuggestion(group: string, conflictsLength: number) {
-  if (group === 'tags') {
-    process.stderr.write(green(`
-    ${conflictsLength} conflict(s) on tags.
-    Suggestion: please use ${blue('prefix-tags-with-filename')} or ${blue('prefix-tags-with-info-prop')} to prevent naming conflicts. \n\n`
-    ));
+function prefixTagSuggestion(conflictsLength: number, skipTagsCheck?: boolean) {
+  if (!skipTagsCheck) {
+    process.stderr.write(
+      green(`
```

Comment:
Do we use something like `outdent` or `dedent`?

Reply:
Thx, let's do it, this is make sense for me!